### PR TITLE
Replace possibly faulty UN units with TPGM

### LINKED ISSUE #1319: Rebel garrison FFs on spawn
**Describe the bug**
Garrison troops fight eachother on spawning.

**To Reproduce**
Steps to reproduce the bug (If you can't reproduce it, just give details of what happened beforehand. Anything you think could be relevant):
Spawn garrison troops on 3CB Blufor

**Expected behaviour**
Them to not infight
**Screenshots**
See https://discordapp.com/channels/449481990513754112/647694189236322305/735521836208750752
**Environment (please complete the following information):**
Version: 2.3
Map (specify if Altis Blufor): AltisB

[  ] LAN Hosted
[  ] Private dedicated server
[  ] Private dedicated server with Headless client(s)
[  ] Community Server 1
[  ] Community Server 2
[  ] Community Test Server
[  ] Community Test Server 2

Modset: 3CB
    
Time the bug occurred (Please leave in GMT+0):
If **Not** on a community server, please leave the time zone of your client for LAN, or the server for dedicated: 

**Additional context**

If you were LAN hosting, please attach your client .RPT file from when the bug occurred, If on a private dedicated server, please attach the server's .RPT from when the bug occurred.

### PR DESCRIPTION
**Pull request by SonorousMeerkat**
Someone reported FF in the AI, all are now different and safe.

## What type of PR is this.
1. [x] Bug
2. [x] Change
3. [ ] Enhancement

### What have you changed and why?
Information:
    All units in 3CB Reb TPGM Arid replaced from UN to TPGM

### Please specify which Issue this PR Resolves.
closes #1319 

### Please verify the following and ensure all checks are completed.

1. [ ] Have you loaded the mission in singleplayer?
2. [ ] Have you loaded the mission in LAN host?
3. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
  load mission, spawn one of each rebel unit + vic
********************************************************
Notes:

### CODE REVIEW

**Comment by Lazejun** [comment]
TKM vs TKM here we go bois 👀

I will take a look when I am home later today but I Doubt changing Bluefor units with other unit ends the issue

**Comment by SonorousMeerkat** [comment]
in editor, they are TPGM.

**Comment by SonorousMeerkat** [comment]
Blufor won't be around much longer anyway...

File: A3-Antistasi/Templates/3CB_Reb_TPGM_Arid.sqf [hunk 1]
```diff
@@ -4,23 +4,23 @@
 nameTeamPlayer = "UN";
```
Comment (Lazejun) [inline A3-Antistasi/Templates/3CB_Reb_TPGM_Arid.sqf:4]: Wouldnt it make sense to change this too?

Comment (SonorousMeerkat) [reply A3-Antistasi/Templates/3CB_Reb_TPGM_Arid.sqf]: done

File: A3-Antistasi/Templates/3CB_Reb_TPGM_Arid.sqf [hunk 2]
```diff
@@ -41,17 +41,17 @@ soldiersSDK = sdkTier1 + sdkTier2 + sdkTier3;
 
 ////////////////////////////////////
 //           VEHICLES            ///
-////////////////////////////////////
+////////////////////////////////////"","","","","","",""
```
Comment (Lazejun) [inline A3-Antistasi/Templates/3CB_Reb_TPGM_Arid.sqf:44]: What happend here?

File: A3-Antistasi/Templates/3CB_Reb_TPGM_Arid.sqf [hunk 3]
```diff
@@ -41,17 +41,17 @@ soldiersSDK = sdkTier1 + sdkTier2 + sdkTier3;
...
 //Military Vehicles
 vehSDKBike = "B_G_Quadbike_01_F";
-vehSDKLightArmed = "UK3CB_UN_B_Hilux_Pkm";
-vehSDKAT = "UK3CB_UN_B_LR_SPG9";
-vehSDKLightUnarmed = "UK3CB_UN_B_Hilux_Closed";
-vehSDKTruck = "UK3CB_UN_B_V3S_Closed";
+vehSDKLightArmed = "UK3CB_TKM_B_Hilux_Pkm";
+vehSDKAT = "UK3CB_TKM_B_Hilux_Rocket";//Don't worry about it.
```
Comment (Lazejun) [inline A3-Antistasi/Templates/3CB_Reb_TPGM_Arid.sqf:48]: You know that it comes with 3 Magazines of 19 times 70 mm Rockets that can instantly Destroy the Heaviest Tanks of TKM/TKA (T55).

3 AT Cars for TKM ["UK3CB_TKM_I_Hilux_Spg9","UK3CB_TKM_I_LR_SPG9","UK3CB_TKM_I_UAZ_SPG9"]
You got Plenty of Not broken and overpowered choices.

File: A3-Antistasi/Templates/3CB_Reb_TPGM_Arid.sqf [hunk 4]
```diff
@@ -1,26 +1,26 @@
 ////////////////////////////////////
 //       NAMES AND FLAGS         ///
 ////////////////////////////////////
-nameTeamPlayer = "UN";
+nameTeamPlayer = "TPGM";
 SDKFlag = "Flag_CCM_B";
```
Comment (Lazejun) [inline A3-Antistasi/Templates/3CB_Reb_TPGM_Arid.sqf:5]: Flag_TKM_B
Would be the Flag
Texuture:
"uk3cb_factions\addons\uk3cb_factions_tkm\flag\tkm_b_flag_co.paa"

File: A3-Antistasi/Templates/3CB_Reb_TPGM_Arid.sqf (hunk 2 above)
Comment (SonorousMeerkat) [reply A3-Antistasi/Templates/3CB_Reb_TPGM_Arid.sqf:44]: oops, artefact of replacing things.

File: A3-Antistasi/Templates/3CB_Reb_TPGM_Arid.sqf (hunk 3 above)
Comment (SonorousMeerkat) [reply A3-Antistasi/Templates/3CB_Reb_TPGM_Arid.sqf:48]: Fine...

File: A3-Antistasi/Templates/3CB_Reb_TPGM_Arid.sqf (hunk 4 above)
Comment (SonorousMeerkat) [reply A3-Antistasi/Templates/3CB_Reb_TPGM_Arid.sqf:5]: done

**Pull request merged**